README.md

```diff
@@ -20,8 +20,7 @@ The `/search` API endpoint has few additional query parameters. More might be ad
   a package requires 7.4, the package will not be returned or an older compatible package will be shown.
   By default this endpoint always returns only the newest compatible package.
 * category: Filters the package by the given category. Available categories can be seend when going to `/categories` endpoint.
-* package: Filters by a specific package name, for example `mysql`. In contrast to the other endpoints, it will return
-  by default all versions of this package.
+* package: Filters by a specific package name, for example `mysql`. It will return the most recent version of the package.
```

Comment:
i think you meant to put this in the search PR.  I was referring to updating the API endpoint path one line 12

Reply:
@neptunian argh, sorry. Now did the right adjustement.